apps/docs/src/app/platform/component-docs/platform-multi-combobox/platform-multi-combobox-docs.module.ts

```diff
@@ -45,7 +46,10 @@ const routes: Routes = [
         BusyIndicatorModule
     ],
     exports: [RouterModule],
-    providers: [{ provide: MOBILE_MODE_CONFIG, useValue: MULTI_COMBOBOX_MOBILE_CONFIG, multi: true }],
+    providers: [
+        { provide: MOBILE_MODE_CONFIG, useValue: MULTI_COMBOBOX_MOBILE_CONFIG, multi: true },
+        { provide: MAP_LIMIT, useValue: 12 }
```

Comment:
since you're using the factory as @g-cheishvili suggested, you don't need to provide it anymore

Reply:
@dmitry-stepanenko done